addons/board/static/src/add_to_board/add_to_board.js

```diff
@@ -87,7 +87,7 @@ AddToBoard.template = "board.AddToBoard";
 const addToBoardItem = {
     Component: AddToBoard,
     groupNumber: 4,
-    isDisplayed: ({ config }) => config.actionType === "ir.actions.act_window",
+    isDisplayed: ({ config }) =>{return config.actionType === "ir.actions.act_window" && config.actionId;} ,
```

Comment:
```suggestion
    isDisplayed: ({ config }) => config.actionType === "ir.actions.act_window" && config.actionId,
```
this should be equivalent